src/ui/public/index_patterns/static_utils/index.js

```diff
@@ -0,0 +1,7 @@
+export function getFieldByName(fields, name) {
+  return fields.find(field => field.name === name);
+}
+
+export function isFilterable(field) {
+  return ['string', 'number', 'date', 'ip', 'boolean'].includes(field.type);
```

Comment:
Could you use [this](https://github.com/elastic/kibana/blob/d9d9fb22fb0019e6090cd39b8fb3d13fc92050a9/src/utils/kbn_field_types.js#L20) to determine if it's filterable instead of listing them out here?

Reply:
That's a good idea! So if `KBN_FIELD_TYPES ` is exported I can do like:

```js
const isFilterable = field => 
  KBN_FIELD_TYPES.some(type => type.name === field && type.filterable)
```